plugins/destination/mysql/client/transformer.go

```diff
@@ -0,0 +1,166 @@
+package client
+
+import (
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func (*Client) TransformBool(v *schema.Bool) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Bool
+}
+
+func (*Client) TransformBytea(v *schema.Bytea) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Bytes
+}
+
+func (*Client) TransformFloat8(v *schema.Float8) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Float
+}
+
+func (*Client) TransformInt8(v *schema.Int8) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Int
+}
+
+func (*Client) TransformTimestamptz(v *schema.Timestamptz) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Time
+}
+
+func (*Client) TransformJSON(v *schema.JSON) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+
+	return string(v.Bytes)
```

Comment:
unescape logic check (see mssql implementation & required transformation + https://github.com/cloudquery/plugin-sdk/issues/622)

Reply:
Fixed in https://github.com/cloudquery/plugin-sdk/pull/714